frontend/packages/kubevirt-plugin/integration-tests-cypress/view/dialog.ts

```diff
@@ -0,0 +1,57 @@
+import { Disk, Network } from '../types/vm';
+import { diskDialog, nicDialog } from './selector';
+
+export const dialog = {
```

Comment:
The console team is exporting namespaces in their view files to export related items from a file separately from other items in the file. I think this is a convention we should use instead of putting everything into object literals. Below is an example of how they handle selectors in their details page view file. The enums we use work well enough for selectors alone, but I think we could use it to organize functions and anything else that the enums don't work for.

```javascript
export namespace DetailsPageSelector {
  export const name = 'dd[data-test-selector="details-item-value__Name"]';
  export const namespace = 'dd[data-test-selector="details-item-value__Namespace"] a';
  export const sectionHeadings = '[data-test-section-heading]';
  export const itemLabels = 'dt';
  export const horizontalNavTabs = '.co-m-horizontal-nav__menu-item';
}
```

Reply:
move these functions out of the object.